src/sage/rings/polynomial/complex_roots.py

```diff
@@ -46,6 +46,7 @@
 from sage.rings.qqbar import AA, QQbar
 from sage.arith.misc import sort_complex_numbers_for_display
 from sage.rings.polynomial.refine_root import refine_root
+from sage.rings.polynomial.polynomial_ring_constructor import PolynomialRing
```

Comment:
```suggestion
```
My guess is this is causing the circular imports. I would make these imports locally.